<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1120</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(245, 245, 245)
}untitled</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="download_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(52, 59, 41);
	color: rgb(255, 255, 255);
	border-radius: 7px;
	font-size: 20px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}</string>
    </property>
    <property name="text">
     <string>Загрузить данные по финансовым расходам</string>
    </property>
   </widget>
   <widget class="QLabel" name="enter_exel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size: 20px;
	color: rgb(65, 74, 76);
}</string>
    </property>
    <property name="text">
     <string>Ввести данные по финанасовым расходам:</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="sum_lbl">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font-size: 20px;
	color: rgb(65, 74, 76);
}</string>
       </property>
       <property name="text">
        <string>Сумма:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dcribe_lbl">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font-size: 20px;
	color: rgb(65, 74, 76);
}</string>
       </property>
       <property name="text">
        <string> Описание:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="date_lbl">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font-size: 20px;
	color: rgb(65, 74, 76);
}</string>
       </property>
       <property name="text">
        <string>       Дата:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="sum_line">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(52, 59, 41);
	color: rgb(255, 255, 255);
	font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>|</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dscribe_line">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(52, 59, 41);
	color: rgb(255, 255, 255);
	font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>|</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="date_line">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(52, 59, 41);
	color: rgb(255, 255, 255);
	font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>__.__.____</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download_btn_">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(52, 59, 41);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 16px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}</string>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLabel" name="enter_exel_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size: 17px;
	color: rgb(65, 74, 76);
}</string>
    </property>
    <property name="text">
     <string>Расходы и доходы за последнее время:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>320</y>
      <width>16</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Line{
	color: rgb(65, 74, 76);
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>191</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="spnt_lbl_1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="spnt_lbl_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="spnt_lbl_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="spnt_lbl_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="spnt_lbl_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="spnt_lbl_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="spnt_lbl_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>21</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="redact_btn_1">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(33, 161, 189);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 16px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}</string>
       </property>
       <property name="text">
        <string>✎</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="redact_btn_2">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(33, 161, 189);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 16px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}</string>
       </property>
       <property name="text">
        <string>✎</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="redact_btn_3">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(33, 161, 189);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 16px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}</string>
       </property>
       <property name="text">
        <string>✎</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="redact_btn_4">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(33, 161, 189);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 16px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}</string>
       </property>
       <property name="text">
        <string>✎</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="redact_btn_5">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(33, 161, 189);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 16px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}</string>
       </property>
       <property name="text">
        <string>✎</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="redact_btn_6">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(33, 161, 189);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 16px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}</string>
       </property>
       <property name="text">
        <string>✎</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="redact_btn_7">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(33, 161, 189);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 16px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}</string>
       </property>
       <property name="text">
        <string>✎</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>24</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="del_btn_1">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(242, 71, 38);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 16px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QPushButton&quot; name=&quot;del_btn_1&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;450&lt;/x&gt;
     &lt;y&gt;340&lt;/y&gt;
     &lt;width&gt;19&lt;/width&gt;
     &lt;height&gt;19&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;styleSheet&quot;&gt;
    &lt;string notr=&quot;true&quot;&gt;QPushButton{
	background-color: rgb(242, 71, 38);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 10px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}&lt;/string&gt;
   &lt;/property&gt;
   &lt;property name=&quot;text&quot;&gt;
    &lt;string&gt;🗑&lt;/string&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
       </property>
       <property name="text">
        <string>🗑</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="del_btn_2">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(242, 71, 38);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 16px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QPushButton&quot; name=&quot;del_btn_1&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;450&lt;/x&gt;
     &lt;y&gt;340&lt;/y&gt;
     &lt;width&gt;19&lt;/width&gt;
     &lt;height&gt;19&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;styleSheet&quot;&gt;
    &lt;string notr=&quot;true&quot;&gt;QPushButton{
	background-color: rgb(242, 71, 38);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 10px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}&lt;/string&gt;
   &lt;/property&gt;
   &lt;property name=&quot;text&quot;&gt;
    &lt;string&gt;🗑&lt;/string&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
       </property>
       <property name="text">
        <string>🗑</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="del_btn_3">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(242, 71, 38);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 16px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QPushButton&quot; name=&quot;del_btn_1&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;450&lt;/x&gt;
     &lt;y&gt;340&lt;/y&gt;
     &lt;width&gt;19&lt;/width&gt;
     &lt;height&gt;19&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;styleSheet&quot;&gt;
    &lt;string notr=&quot;true&quot;&gt;QPushButton{
	background-color: rgb(242, 71, 38);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 10px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}&lt;/string&gt;
   &lt;/property&gt;
   &lt;property name=&quot;text&quot;&gt;
    &lt;string&gt;🗑&lt;/string&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
       </property>
       <property name="text">
        <string>🗑</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="del_btn_4">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(242, 71, 38);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 16px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QPushButton&quot; name=&quot;del_btn_1&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;450&lt;/x&gt;
     &lt;y&gt;340&lt;/y&gt;
     &lt;width&gt;19&lt;/width&gt;
     &lt;height&gt;19&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;styleSheet&quot;&gt;
    &lt;string notr=&quot;true&quot;&gt;QPushButton{
	background-color: rgb(242, 71, 38);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 10px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}&lt;/string&gt;
   &lt;/property&gt;
   &lt;property name=&quot;text&quot;&gt;
    &lt;string&gt;🗑&lt;/string&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
       </property>
       <property name="text">
        <string>🗑</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="del_btn_5">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(242, 71, 38);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 16px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QPushButton&quot; name=&quot;del_btn_1&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;450&lt;/x&gt;
     &lt;y&gt;340&lt;/y&gt;
     &lt;width&gt;19&lt;/width&gt;
     &lt;height&gt;19&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;styleSheet&quot;&gt;
    &lt;string notr=&quot;true&quot;&gt;QPushButton{
	background-color: rgb(242, 71, 38);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 10px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}&lt;/string&gt;
   &lt;/property&gt;
   &lt;property name=&quot;text&quot;&gt;
    &lt;string&gt;🗑&lt;/string&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
       </property>
       <property name="text">
        <string>🗑</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="del_btn_6">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(242, 71, 38);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 16px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QPushButton&quot; name=&quot;del_btn_1&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;450&lt;/x&gt;
     &lt;y&gt;340&lt;/y&gt;
     &lt;width&gt;19&lt;/width&gt;
     &lt;height&gt;19&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;styleSheet&quot;&gt;
    &lt;string notr=&quot;true&quot;&gt;QPushButton{
	background-color: rgb(242, 71, 38);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 10px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}&lt;/string&gt;
   &lt;/property&gt;
   &lt;property name=&quot;text&quot;&gt;
    &lt;string&gt;🗑&lt;/string&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
       </property>
       <property name="text">
        <string>🗑</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="del_btn_7">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(242, 71, 38);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 16px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QPushButton&quot; name=&quot;del_btn_1&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;450&lt;/x&gt;
     &lt;y&gt;340&lt;/y&gt;
     &lt;width&gt;19&lt;/width&gt;
     &lt;height&gt;19&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;styleSheet&quot;&gt;
    &lt;string notr=&quot;true&quot;&gt;QPushButton{
	background-color: rgb(242, 71, 38);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 10px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}&lt;/string&gt;
   &lt;/property&gt;
   &lt;property name=&quot;text&quot;&gt;
    &lt;string&gt;🗑&lt;/string&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
       </property>
       <property name="text">
        <string>🗑</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="all_spents_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>540</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(52, 59, 41);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 16px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}</string>
    </property>
    <property name="text">
     <string>Все траты</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="graph_main_pg" native="true">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>80</y>
      <width>511</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">PlotWidjet{
	Background-color: rgb(255, 255, 255)
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="all_graph_btn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>440</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(52, 59, 41);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	font-size: 16px;
}
QPushButton:pressed {
	background-color: rgb(39, 45, 30);
}</string>
    </property>
    <property name="text">
     <string>График за всё время</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
